nifi-nar-bundles/nifi-extension-utils/nifi-event-transport/src/main/java/org/apache/nifi/event/transport/netty/NettyEventSenderFactory.java

```diff
@@ -49,6 +49,7 @@
  */
 public class NettyEventSenderFactory<T> extends EventLoopGroupFactory implements EventSenderFactory<T> {
     private static final int MAX_PENDING_ACQUIRES = 1024;
+    private final static int FRAME_SIZE = 40000;
```

Comment:
Is there a particular reason for this specific frame size?  It seems like it should be a configurable `Integer` property that could be checked for null and used to set the Send and Receive Buffer sizes when found.